AspNetCore.sln

```diff
@@ -6353,11 +6353,13 @@ Global
 		{F8AB0D33-1082-4F41-90E7-77C06EDD983D}.Release|x86.ActiveCfg = Release|Any CPU
 		{F8AB0D33-1082-4F41-90E7-77C06EDD983D}.Release|x86.Build.0 = Release|Any CPU
 		{647F0202-165F-4E98-A798-B2B3113050EC}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{647F0202-165F-4E98-A798-B2B3113050EC}.Debug|Any CPU.Build.0 = Debug|Any CPU
```

Comment:
The lack of these lines meant this project (IISSamples) wasn't actually marked to build.